### DISCUSSION TITLE
设置ipv6-support为off仍然监听IPv6地址，通过服务器IP:2017无法访问网页GUI

### DISCUSSION THREAD

**Discussion by 10wei1000** [v2rayA/v2rayA#1485] [Q&A]
参考官方文档在/etc/systemd/system/v2raya.service.d/xray.conf中添加了：

[Service]
Environment="V2RAYA_IPV6_SUPPORT=off"

通过 systemd 服务文件设置启动参数：
sudo systemctl edit --full v2raya.service
修改
ExecStart=/usr/bin/v2raya --log-disable-timestamp --address 0.0.0.0:2017 --ipv6-support off
也不行。

监听的类型还是IPv6：

root@ht2024021707943:~# sudo lsof -i:2017
COMMAND   PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
v2raya  22274 root    9u  IPv6 147117      0t0  TCP *:2017 (LISTEN)

系统为Ubuntu-20.04.1-x64

烦请大神赐教，不胜感激！

**Comment by tcpqueue** [comment]
同问
